### ISSUE TITLE
New-BCImage does not work on PublicPreview

### ISSUE THREAD

**Issue by phenno1** [microsoft/navcontainerhelper#2369]
**Describe the issue**
When creating new BC Image for PublicPreview we get an Error that SQL Service cannot be started. Same error appears if we try to create new container using mentioned artifacts. 

**Scripts used to create container and cause the issue**
```
New-BcImage -artifactUrl (Get-BCArtifactUrl -storageAccount BcPublicPreview) -imageName w1:20
```

**Full output of scripts**
```
w1:20
Building image w1:20 based on mcr.microsoft.com/businesscentral:10.0.17763.2686 with https://BcPublicPreview.azureedge.net/sandbox/20.0.37118.0/w1
Pulling latest image mcr.microsoft.com/businesscentral:10.0.17763.2686
10.0.17763.2686: Pulling from businesscentral
Digest: sha256:6116cdec65e9a3f6d41ba47319d296c1c6b42395327ffdffc9b65b0005823176
Generic Tag: 1.0.2.1
Container OS Version: 10.0.17763.2686 (ltsc2019)
Host OS Version: 10.0.17763.379 (ltsc2019)
WARNING: Host OS and Base Image Container OS doesn't match and Hyper-V is not installed. If you encounter issues, you could try to install Hyper-V.
Using process isolation
Files in c:\bcartifacts.cache\5wlfudyb.tyk\my:
Copying Platform Artifacts
c:\bcartifacts.cache\sandbox\20.0.37118.0\platform
Copying Database
Copying Licensefile
Copying Extensions
C:\bcartifacts.cache\sandbox\20.0.37118.0\w1\Extensions
c:\bcartifacts.cache\5wlfudyb.tyk
Sending build context to Docker daemon  1.984GB

Step 1/6 : FROM mcr.microsoft.com/businesscentral:10.0.17763.2686
 ---> 6f30fec3aafa
Step 2/6 : ENV DatabaseServer=localhost DatabaseInstance=SQLEXPRESS DatabaseName=CRONUS IsBcSandbox=Y artifactUrl=https://BcPublicPreview.azureedge.net/sandbox/20.0.37118.0/w1 filesOnly=False
 ---> Running in fba04b0f7df0
Removing intermediate container fba04b0f7df0
 ---> 955a6c3a341d
Step 3/6 : COPY my /run/
 ---> fd7a41d973f2
Step 4/6 : COPY NAVDVD /NAVDVD/
 ---> bd20368db00d
Step 5/6 : RUN \Run\start.ps1 -installOnly -multitenant
 ---> Running in a1c396b666fc
Using installer from C:\Run\150-new
Installing Business Central
Installing from DVD
Starting Local SQL Server
Service 'SQL Server Browser (SQLBrowser)' cannot be started due to the following error: Cannot start service SQLBrowser on computer '.'.
Installation failed
At C:\Run\start.ps1:372 char:9
+         throw "Installation failed"
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Installation failed:String) [
   ], RuntimeException
    + FullyQualifiedErrorId : Installation failed

New-BcImage Telemetry Correlation Id: cd197d45-2c8b-4fc6-ba20-b584c9a70e84
Docker Build failed with exit code 1
    + CategoryInfo          : OperationStopped: (Docker Build failed with exit code 1:String) [], RuntimeException
    + FullyQualifiedErrorId : Docker Build failed with exit code 1

```

**Additional context**
Few days ago we created new container successfully but in that container image was: mcr.microsoft.com/businesscentral:10.0.17763.437

Now when we try to create new BC image or container for BCPublicPreview we get: mcr.microsoft.com/businesscentral:10.0.17763.2686

**Comment by freddydk** [comment]
You are running an un-updated WIndows 2019 server on the host.
If you run Windows update - it will likely work again (like https://github.com/microsoft/navcontainerhelper/issues/2350)
You can also specify a specific generic image if you for some reason do not want to update the host.
Please note though that Microsoft highly recommends that windows are kept up-to-date in order to be safe.

**Comment by phenno1** [comment]
Dear Freddy, 

thank you very much. It could be due to OS updates and I will check what can be done about that. I guess we can close this issue (was trying to find similar issues before but didn't bump to #2350).

**Comment by freddydk** [comment]
fixed in 3.0.4 - it will locate a better generic image
